Clarify Chef Infra Client event vs. eventlogger (#3277)
chef/chef-web-docs: content/config_rb_client.md

: The maximum size (in bytes) of a diff file Chef Infra Client can create. Default value: `1000000`.

`disable_event_logger`

> : Enable or disable sending Chef Infra Client internal state events to the Microsoft Windows "Application" event log. Set to `false` to send events to the Microsoft Windows "Application" event log at the start and end of a Chef Infra Client run, and also if a Chef Infra Client run fails. This setting works in combination with `log_location` below. See `log_location` if you need to change the ultimate destination of your Chef Infra Client logs to the Windows Eventlog.  Set to `true` to disable event logging. Default value: `false`.

Reviewer: ```suggestion
: Enable or disable sending Chef Infra Client internal state events to the Microsoft Windows "Application" event log. Set to `false` to send events to the Microsoft Windows "Application" event log at the start and end of a Chef Infra Client run, and also if a Chef Infra Client run fails. Use `log_location` to set the destination of your Chef Infra Client logs to the Windows event log. Set to `true` to disable event logging. Default value: `false`.
```
